adds beginning css

he8079 opened the pull request:
more info on flexbox for the menu and footer needed.

he8079 committed:
adds beginning css

more info on flexbox for the menu and footer needed.

he8079 committed:
added flexbox footer/update

Also added flex to header/nav, but not finished styling yet. Added some JS to get nav to toggle.

he8079 committed:
added testimonial scrolling box

also added style to header. Now the dropdown menu is behind the testimonial box; how do I fix this?

he8079 committed:
Update css z-index

Updated z-index on dropdown menu to put above testimonial box

axamei commented on index.html:
```diff
@@ -4,44 +4,111 @@
-    <div> 
-        <h1>Testimonials</h1>
-        <p>Testimonial from a client</p>
+   <div class="slideshow-container">
```
missing an h1 on this page. you can hide it visually if you don't want it displayed, but it needs to exist. see https://www.a11yproject.com/posts/how-to-hide-content/

he8079 replied:
Fixed h1 on index page.

axamei commented on index.html:
```diff
@@ -4,44 +4,111 @@
     <div>
-        <h2>Contact Us</h2>
-        <p>moorestudentsuccess@gmail.com</p>
+        <table class="indextable">
```
tables should only be used as a data table / comparing data. use divs and flexbox for this layout

he8079 replied:
The table is only a placeholder and I will be changing it this week. I wanted to try making a basic table to learn how. I never intended it to be permanent.

he8079 replied:
table removed and updated with div boxes.

he8079 committed:
Added divs to footer and index page

Added divs to footer. Redesigned index page and added div boxes for services. Removed table. Slide show on about page in process- it is not finished.

axamei reviewed (approved):
👍
